.github/workflows/main.yaml

```diff
@@ -94,8 +94,6 @@ jobs:
         uses: canonical/charming-actions/channel@2.2.5
         id: channel
       - name: Upload charm to Charmhub
-        uses: canonical/charming-actions/upload-charm@2.2.5
```

Comment:
We could increase the functionality of the [upload-charm](https://github.com/canonical/charming-actions/tree/main/upload-charm) action and make it support a `built-charm` parameter so that it doesn't build the charm.

Reply:
Yes, I agree that we should probably work a bit more on the workflow and make it a shared workflow that is easy to reuse. It could also just mean updating the `upload-charm` action to support providing it a built charm. I can add this to the agenda for the next charm guild meeting.